docs/DECISIONS.md

```
@@ -1,5 +1,8 @@
 # Decisions Log
 
+# Decisions Log
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)